### DISCUSSION TITLE
Emscripten or WASI support

### DISCUSSION THREAD

**Discussion by sachitv** [ihedvall/mdflib#66] [Ideas]
Hi, thanks for the amazing work with this library. I was wondering whether you had any plans to officially add [Emscripten](https://emscripten.org/) or [WASI](https://wasi.dev/) support? I ask because I was interested in building some web applications that can work with MF4 files. Would you otherwise be open to contributions for the same?

**Comment by ihedvall** [comment]
I am only using the C++ version in my other applications. I decided early to do binding to other "languages" as WebAssembly/JS, on request. Currently it exist a .NET assembly and I am currently working on the Python Interface.

If you want a WebAssembly interface, there should not be any problem to doing that. I have made a project for MDF application and task. I recommend to use that project for planning and setting up requirements. I recommend an on-line meeting before starting any work.

Note that my other applications/services, uses the Google RPC as standard interface.

If it is OK with you, I transfer this issue to the discussion area.
